provider-ui/components/Provider.js

```diff
@@ -0,0 +1,260 @@
+import React, { Fragment, useEffect, useState } from "react";
```

Comment:
The tabwidth used in this file is 4 while in other it is 2. 
Can you please enforce 2, all around the project via eslintrc.